torch/ao/quantization/observer.py

```diff
@@ -1539,7 +1551,7 @@ def load_observer_state_dict(mod, obs_dict):
 """
 
 default_dynamic_quant_observer = PlaceholderObserver.with_args(
-    dtype=torch.float, compute_dtype=torch.quint8
+    dtype=torch.quint8, compute_dtype=torch.quint8
```

Comment:
do we need `is_dynamic` here?

Reply:
oh, saw this in the summary, so now we are still using `dtype` and `compute_dtype`, we'll use `dtype` and `is_dynamic` in a future PR